lib/controller/LedenLijstController.php

```diff
@@ -50,16 +50,16 @@ public function lijst(
 					if ($query['status'] == 'LEDEN') {
 						$query['status'] = 'LEDEN|OUDLEDEN';
 						$message =
-							'Zoekterm gaf geen resultaten met gegeven statusfilter, gezocht in <em>leden &amp; oudleden</em>.';
+							'Zoekterm gaf geen resultaten met gegeven statusfilter, gezocht in **leden & oudleden**. Om alle leden en novieten weer te geven zoek met \'%\'.';
 					} elseif ($query['status'] == 'LEDEN|OUDLEDEN') {
 						$query['status'] = 'ALL';
 						$message =
-							'Zoekterm gaf geen resultaten met gegeven statusfilter, gezocht in <em>alle leden</em>.';
+							'Zoekterm gaf geen resultaten met gegeven statusfilter, gezocht in **alle leden**. Om alle leden en novieten weer te geven zoek met \'%\'.';
 					}
 				} else {
 					$query['status'] = 'LEDEN|OUDLEDEN';
 					$message =
-						'Zoekterm gaf geen resultaten met gegeven statusfilter, gezocht in <em>leden &amp; oudleden</em>.';
+						'Zoekterm gaf geen resultaten met gegeven statusfilter, gezocht in **leden &amp; oudleden**. Om alle leden en novieten weer te geven zoek met \'%\'.';
```

Comment:
Als we dit hier opschrijven kunnen we het net zo goed de default maken. Hij zocht niet automatisch in iedereen om het zoeken op iedereen niet aan te moedigen ivm privacy enzo @CodexNine

Reply:
Ho ik bedoel @cbasje

Reply:
Ja ik snap wel dat het zo is, maar het voelt gek dat je op een 'Ledenlijst' klikt maar dat je dan alsnog moet zoeken. Ik denk ook dat het sws beter is om minder velden automatisch te selecteren want nu geeft ie meteen telefoonnummer en emailadres van iedereen. Ik ben het met je eens dat het dan eigenlijk gewoon dan default iedereen moet laten zien, maar deze pull request was eigenlijk meer bedoeld om 'adres_ouders' toe te voegen voor de P. De rest kan ik later bekijken.